### ISSUE TITLE
Unable to fetch archives, need to run update with --fix-missing

### ISSUE THREAD

**Issue by JoeNunes** [tensorflowRUs/ai-environment-setup#1]
Copy of session attached, 2nd attempt, same result.
[vbvm build 2nd attempt.txt](https://github.com/tensorflowRUs/ai-environment-setup/files/4171598/vbvm.build.2nd.attempt.txt)

**Comment by zoetsekas** [comment]
are you behind a corp proxy?

**Comment by JoeNunes** [comment]
No, just wifi to standard home modem, standard config. Windows 10 enterprise 64 bit.

**Comment by JoeNunes** [comment]
3rd attempt with webroot realtime and firewall protection off, errors on different lines, seems random.
[vbvm build 3rd attempt.txt](https://github.com/tensorflowRUs/ai-environment-setup/files/4173209/vbvm.build.3rd.attempt.txt)
